src/js/components/recipient/topFive/TopFive.jsx

```diff
@@ -62,7 +62,7 @@ export default class TopFive extends React.Component {
         let disclaimer;
         if (this.props.category === 'country' || this.props.category === 'state_territory') {
             disclaimer = (
-                <span> <br /><strong>Note: </strong> <em> This data is based on Primary Place of Performance.</em> </span>
+                <sub> <strong>Note: </strong> <em> This data is based on Primary Place of Performance.</em> </sub>
```

Comment:
Could you use CSS to decrease the font size instead of subscript? `<sub>` is intended for text that should appear below the normal line (e.g. H<sub>2</sub>O)